packages/amplify-util-mock/src/api/api.ts

```diff
@@ -118,7 +119,7 @@ export class APITest {
       }
     } catch (e) {
       // failed to stop opensearch emulator
-      context.print.error(`Failed to stop OpenSearch Local Server ${e.message}`);
+      printer.error(`Failed to stop OpenSearch Local Server ${e.message}. Please kill the mock process and restart it.`);
```

Comment:
```suggestion
      printer.error(`Failed to stop OpenSearch Local Server ${e.message}. Kill the mock process and restart it.`);
```
Additionally, is there any way we could get the PID of the process so we could print `Kill the process using "kill -9 <PID>"`